src/Core/Components/Switch/FluentSwitch.razor

```diff
@@ -0,0 +1,19 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@inherits FluentInputBase<bool>
+
+<FluentField InputComponent="@this" ForId="@Id" Class="@ClassValue" Style="@StyleValue">
+    <fluent-switch @ref="@Element"
+                     aria-label="@AriaLabel"
```

Comment:
Can you align all attributes with previous?

Reply:
ok, attributes aligned.